mmm/describe/describe.py

```diff
@@ -467,13 +468,33 @@ def describe_mmm_training(
     )
     cost_per_target_hat = 1.0 / roi_hat
 
+    media_effect_df = get_media_effect_df(data_to_fit, media_effect_hat)
+
     _dump_posterior_metrics(
         results_dir,
-        get_media_effect_df(data_to_fit, media_effect_hat),
+        media_effect_df,
         get_roi_df(data_to_fit, media_effect_hat, roi_hat),
         get_cost_per_target_df(data_to_fit, media_effect_hat, roi_hat, cost_per_target_hat),
     )
 
+    try:
+        medians = media_effect_df["median"]
+        blended_median = medians.get("blended")
+        top_medians = medians.drop("blended").sort_values(ascending=False).head(3)
+
+        summary = {
```

Comment:
Love it.

I _think_ it would be better to do this in the kube job, i.e. consuming the CSV file after running the model finishes.  But that's probably worth discussing and also not something we need to do right now.

Reply:
I'm thinking processing raw data is usually better than some other output file, but there might be other considerations that I haven't really thought of here. We can iterate on this down the line 👍